v3/plugins/eclipse/che-theia/7.1.0/meta.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: v2
+publisher: eclipse
+name: che-theia
+version: 7.1.0
+type: Che Editor
+displayName: theia-ide
+title: Eclipse Theia development version.
+description: Eclipse Theia, get the latest release each day.
+icon: https://raw.githubusercontent.com/theia-ide/theia/master/logo/theia-logo-no-text-black.svg?sanitize=true
+category: Editor
+repository: https://github.com/eclipse/che-theia
+firstPublicationDate: "2019-03-07"
```

Comment:
I would say that it's today the first publication date for che-theia 7.1.0

Reply:
good point, but I believe we have not updated the `firstPublicationDate` before and it might be assumed that `firstPublicationDate` = firstPublicationDate of plugin in general, but I'm not sure